gatsby-node.js

```diff
@@ -15,6 +14,10 @@ const PARAMETER_TYPES = {
 const getFormattedPath = (path) => path.replace(/{/g, ':').replace(/}/g, '');
 
 const getAllData = async () => {
+  const dataSwagger = await fetch('https://api.novu.co/api-json')
+    .then((response) => response.json())
+    .then((response) => response);
```

Comment:
יןד 
```suggestion
  const dataSwagger = await fetch('https://api.novu.co/api-json')
    .then((response) => response.json());
```

 I think this should work similarly :) 

And a small question, this will only happen during build right? So we won't fetch this on SSR on reach page visit.

Reply:
@scopsy You are right, thank you!

Yes, this happens during project builds, I would not want to take this to the SSR level as it can significantly affect the page load

I think I can add a webhook to the slack channel where we can trigger a project build at the click of a button - maybe that could simplify use instead of having to go into the Gatsby Cloud every time

what do you think about that?

Reply:
We don't really use slack for the team. But creating a webhoo URL for rebuilds could be great. we can call it from our github actions after deploying production version of the API.